unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Common/Friends/RealmTracker.cs

```diff
@@ -0,0 +1,84 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+class RealmTracker
```

Comment:
Let's explicitly set this `private`. Also, it is strange we are having a private class, what's its purpose?

Reply:
Yeah, this class should be public, I think I forgot set it when I was creating it. Done in https://github.com/decentraland/unity-renderer/pull/1731/commits/5bdcbf932dfd91187bd38aaf4f4339dc4d8022e8